gen/generate.go

```diff
@@ -85,6 +85,9 @@ type Options struct {
 
 	// Name of the file to be generated by ThriftRW.
 	OutputFile string
+
+	//Generate Error on Unknown Enum Marshal
```

Comment:
nit: space after // and the comment should be like a sentence (no need to capitalize every word)

Reply:
```suggestion
	// Generates an error on MarshalText and MarshalJSON if the enum value is
	// unrecognized.
```